### PR TITLE
Fix #1635 - Fixed logged in user dropdown menu easier to click

### LINKED ISSUE #1635: Make items in user dropdown menu easier to click.
The items in the dropdown menus only work if you click the text of the item, and not the entire length and height of the highlighted item when you hover.

This is because each item is a...

```
<li>
  <a>Item Label</a>
</li>
```

...and the padding around the item is applied to the ``<li>`` which isn't clickable.

image

To fix these we either...

* Add the click listener to the ``<li>`` item, or..
* Change the CSS styling so that the ``<a>`` item gets the padding and then makes up the entire clickable area.

### PR DESCRIPTION
**Pull request by raygervais**
This resolve #1635  the issue found [here](https://github.com/mozilla/thimble.mozilla.org/issues/1635) which made the clickable padding of the logged in user menu not register. To remedy, I increased the padding of the link itself and decreased the padding of the `<li>` element which surrounded it.

### CODE REVIEW

File: public/resources/stylesheets/userbar.less [hunk 1]
```diff
@@ -477,7 +477,7 @@ a.navbar-login:hover {
 
       li {
         display: block;
-        padding: 12px;
+        padding: 0px;
```
Comment (gideonthomas) [inline public/resources/stylesheets/userbar.less]: off-hand, let's change this to `0` and remove the `px`

Comment (raygervais) [inline public/resources/stylesheets/userbar.less]: I've fixed the resulting code as per your requested change. If there's anything else let me know

**Comment by flukeout** [comment]
I'll test it out today.

File: public/resources/stylesheets/userbar.less [hunk 2]
```diff
@@ -477,7 +477,7 @@ a.navbar-login:hover {
 
       li {
         display: block;
```
Comment (flukeout) [inline public/resources/stylesheets/userbar.less]: Please remove the ``display: block`` declaration. ``<li>`` items are by definition block-level elements, so we can drop this.

**Comment by raygervais** [comment]
@flukeout, I've fixed your requested change. If you need anything else, let me know.

**Comment by flukeout** [comment]
Tested & works - nice work @raygervais

**Comment by raygervais** [comment]
Thank you @flukeout, I look forward to contributing more to Thimble.

**Pull request merged**